src/css/style.css

```diff
@@ -1,15 +1,99 @@
-.is-vis-hidden{
-    visibility: hidden;
+.is-vis-hidden {
```

Comment:
use https://bulma.io/documentation/helpers/visibility-helpers/ instead

Reply:
@alkeicam  class was not used anywhere, deleted